### ISSUE TITLE
coverage travis job is broken

### ISSUE THREAD

**Issue by saimn** [astropy/ci-helpers#63]
For example: https://travis-ci.org/astropy/astropy/jobs/106803692
```
usage: coveralls [-h] [--verbose] [--dryrun] [--gcov FILE]
                 [--gcov-options GCOV_OPTS] [-r DIR] [-b DIR] [-e DIR|FILE]
                 [-i DIR|FILE] [-E REGEXP] [-x EXT] [-y FILE] [-n] [-t TOKEN]
                 [--encodings ENCODINGS [ENCODINGS ...]] [--dump [FILE]]
                 [--follow-symlinks]

coveralls: error: unrecognized arguments: --merge=c-coveralls.json --rcfile=astropy/tests/coveragerc
```
It seems that the coveralls command here is the one from the cpp-coveralls package (ahah, good joke : https://github.com/eddyxu/cpp-coveralls/blob/master/setup.py#L47), not the one from coveralls.

And the reason why it breaks now is probably #60 ? Comparing with a succesful job,  cpp-coveralls was installed before coverage/coveralls.

**Comment by astrofrog** [comment]
@bsipocz - shall we just move

```
# COVERAGE DEPENDENCIES
if [[ $SETUP_CMD == *coverage* ]]; then
    # TODO can use latest version of coverage (4.0) once astropy 1.1 is out
    # with the fix of https://github.com/astropy/astropy/issues/4175.
    $CONDA_INSTALL coverage==3.7.1
    $PIP_INSTALL coveralls
fi
```

to the end of the build?

**Comment by astrofrog** [comment]
I'll give it a try

**Comment by astrofrog** [comment]
PR in https://github.com/astropy/ci-helpers/pull/64

**Comment by astrofrog** [comment]
Apparently ``cpp-coveralls`` does install a ``coveralls`` command that shadows the real one, so installation order is important: https://github.com/eddyxu/cpp-coveralls/issues/100

**Comment by bsipocz** [comment]
Closing as this was fixed in #64.
